src/libraries/Native/Windows/CMakeLists.txt

```diff
@@ -109,7 +116,7 @@ endif()
 # Debug build specific flags
 list(INSERT __SharedLinkArgs 0 $<$<OR:$<CONFIG:DEBUG>,$<CONFIG:CHECKED>>:/NOVCFEATURE>)
 
-if (${CLR_CMAKE_HOST_ARCH} STREQUAL "x86_64" OR ${CLR_CMAKE_HOST_ARCH} STREQUAL "amd64")
+if (${CLR_CMAKE_HOST_ARCH} STREQUAL "x86_64" OR ${CLR_CMAKE_HOST_ARCH} STREQUAL "amd64" OR ${CLR_CMAKE_HOST_ARCH} STREQUAL "x64")
```

Comment:
Why do we need to add the x64 here?

Reply:
In my local builds the variable was equating to `x64`. The define is not being used in this project currently so it doesn't make a difference at the moment.

Reply:
Hmm, interesting, I wonder if this has other implications. We should check why Windows is getting a different CLR_CMAKE_HOST_ARCH and whether we should fix that (not for this PR, of course)

Reply:
I had briefly looked at it and found this for Windows:
https://github.com/dotnet/runtime/blob/663013d579d5eb3e29b2e0f88aeaed0694e74f39/eng/native/configureplatform.cmake#L233-L235

And this for Linux with separate handling:
https://github.com/dotnet/runtime/blob/663013d579d5eb3e29b2e0f88aeaed0694e74f39/eng/native/configureplatform.cmake#L33-L38
https://github.com/dotnet/runtime/blob/663013d579d5eb3e29b2e0f88aeaed0694e74f39/eng/native/configureplatform.cmake#L220-L222

So probably it would be sufficient to check for `x64`, assuming that `configureplatform.cmake` is also used for this project.

Reply:
I wasn't sure though because `CLR_CMAKE_HOST_ARCH_AMD64` was not defined for this project. I assume the arch is passed separately for this project.